static/css/zamboni/zamboni.css

```diff
@@ -1256,88 +1256,6 @@ ul.license li.copyr { background-position: 0 -260px; }
     color: #390;
 }
 
```

Comment:
I'd think _any_ rule referencing `.persona` is dead wood now.  (not `.persona-license` though, I unwisely re-used it for the static theme license chooser)

Reply:
D'oh!